Source/ACE.Server/WorldObjects/Hook.cs

```diff
@@ -62,7 +62,7 @@ public override ActivationResult CheckUseRequirements(WorldObject activator)
             var rootHouse = House.RootHouse;
             var houseOwner = rootHouse.HouseOwner;
 
-            var houseHooksVisible = rootHouse.HouseHooksVisible ?? true;
+            var houseHooksVisible = rootHouse?.HouseHooksVisible ?? true;
```

Comment:
This check is pointless. The line immediately above where it says rootHouse.HouseOwner will have already crashed.